src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
 // write code here
+const list = document.querySelectorAll('li');
```

Comment:
Ensure that the 'li' elements are present in the DOM when this script runs. If the script is executed before the DOM is fully loaded, 'list' will be empty, and subsequent operations will not work as expected. Consider wrapping the code in a DOMContentLoaded event listener.